src/test/components/elements/EventBookingForm.test.tsx

```diff
@@ -0,0 +1,193 @@
+import { EventBookingForm } from "../../../app/components/elements/EventBookingForm";
+import { renderTestEnvironment } from "../../utils";
+import { mockEvent, mockUser } from "../../../mocks/data";
+import { EmailVerificationStatus, UserContext, UserRole, UserSummaryWithEmailAddressDTO } from "../../../IsaacApiTypes";
+import { API_PATH, augmentEvent } from "../../../app/services";
+import { AdditionalInformation, AugmentedEvent } from "../../../IsaacAppTypes";
+import { Immutable } from "immer";
+import { fireEvent, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { rest } from "msw";
+import * as actions from "../../../app/state/actions";
+
+const requestEmailVerificationSpy = jest.spyOn(actions, "requestEmailVerification");
+const updateAdditionalInformation = jest.fn();
+
+describe("EventBookingForm", () => {
+  const setupTest = ({
+    role,
+    event,
+    user,
+    additionalInformation,
+  }: {
+    role: UserRole;
+    event?: AugmentedEvent;
+    user: typeof mockUser;
```

Comment:
I think this is copy/pasted from somewhere else, as there is a proper type for it now.